trlx/ray_tune/wandb.py

```diff
@@ -0,0 +1,60 @@
+"""Utility function to log the results of a Ray Tune experiment to W&B."""
+
+import os
+import wandb
+import json
+import pandas as pd
+from pathlib import Path
+
+
+def parse_result(result):
+    tmp_result = {}
+    for key, value in result.items():
+        if not "time" in key and isinstance(value, (int, float)):
```

Comment:
this rather specific filter, since these timing variables might also be of interest. can this be changed to something less strict perhaps?

Reply:
yes we can do that. Do you have a suggestion here.